net/pfSense-pkg-pfBlockerNG/Makefile

```diff
@@ -13,6 +13,8 @@ COMMENT=	pfSense package pfBlockerNG
 
 LICENSE=	APACHE20
 
+CONFLICTS=	pfBlockerNG-devel
```

Comment:
The correct package name for conflict is `pfSense-pkg-pfBlockerNG-devel`

Reply:
Fixed